### ISSUE TITLE
Please consider contributing to iraf-community

### ISSUE THREAD

**Issue by olebole** [ysBach/irafdocgen#1]
Hi,

I am writing you on behalf of the IRAF community distribution, https://iraf-community.github.io. The aim of our effort is to save IRAF for future use and reference. A good documentation is one essential requirement for sustainability here.

Therefore it would be great if you could help us maintaining the documentation in a good shape, so that we can offer is on our web page. This would also help bundling forces to keep IRAF alive. The source code of our IRAF distribution is [here](https://github.com/iraf-community/iraf).

What do you think?

Best regards

Ole

**Comment by ysBach** [comment]
Hi! Nice to meet you and thanks for the suggestion :)

I have been aware of your hard working (for maybe at least 5 years?) since last year, and thank you so much for bringing/maintaining IRAF communnity to/at a "visible" area, i.e., github. I am also aware of the IRAF license issue and your effort! :D 

I think it'll be good if the IRAF community offers docs hosting, so basically I agree with your suggestion.

But for the contribution issue, "I" am hesitant about directly contributing to IRAF community. 

FYI: Since I haven't even put a proper ``README`` of this repo yet, which includes the reasons I needed IRAF docs... Those include (1) I need to take good things from it, i.e., benchmarking purpose, for my own packages and lecture materials (because, who can properly read SPP source codes today..?), (2) I wish the astronomical society to detach itself from using IRAF for _professional_ works, but for doing so we need to know how IRAF did the tasks (i.e., docs) so that we can make something at least not worse than IRAF, and (3) so many works have already been published using IRAF, so not having proper docs of IRAF is therefore putting all those works difficult to be understood and validated for all of us and future generations.

I learned IRAF/PyRAF in 2016 and never used it since then because of some critical reasons. Even my home university has never taught it since 2017 and many research groups are discouraging new students from using it, as well as IDL. At the moment, therefore, I am considering to use this repo as _"frozen" docs of IRAF_ which was released in 2013, which is sufficient for my purposes. That is the reason why I copied the IRAF version 2.16.1, not the most recent IRAF community version, as I was worried if any of docs has updated. Also I had to do some complicated works to make parts of the docs into HTML (not commit/push'ed yet) as I am a newbie to HTML, ROFF and even IRAF. So I don't want to test it if the docs are updated, although I am making it to be usable even if some docs undergo minor updates. 

In short, I am not very much motivated in "maintaining" the docs in a "good shape", sorry.

After I update this repo, maybe someone can just copy what I did and make docs from IRAF sources (though it may look very dirty xD). Thus, I think the ones who may want to get docs of more recent IRAF community edition may follow the way I used for this repo, or if you want, maybe that can be used in the IRAF community repo.

**Comment by olebole** [comment]
My motivation for the iraf-community is very similar to your: keep it as a kind of cultural heritage, allowing people to develop alternatives, provide reference calculations for comparison. It is out of question that IRAF is going to be replaced as a whole for good (otherwise I wouldn't volunteer here -- as you I don't use IRAF myself).
The question to contribute is mainly the program that traverses through the sources and creates the HTML documentation pages (optimal would be to do that through Restructured Text btw, since then one could use the advantages of https://readthedocs.org). So it is not that you should maintaining them yourself; I would like to create an environment where this can happen automatically. Do you have something like that? Or was your plan to just keep the sources here?

**Comment by olebole** [comment]
Since I created https://iraf.readthedocs.org a while ago, this issue is outdated (although any help is still appreciated).
